src/Asset/PnpmManager.php

```
@@ -26,6 +28,33 @@ public function isInstalled(): bool
+            '--json',
+            '--audit-level=info',
+            '--lockfile-dir=.',
+            '--ignore-pnpmfile',
+            '--only=null',
+        ];
+
+        if ($noDev) {
+            $command = [...$command, '--prod', '--optional=true'];
+        } else {
+            $command = [...$command, '--prod=false', '--dev=false', '--optional=true'];
+        }
```

PHPMD (2.15.0)

[error] The method getAuditCommand uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)